types/zipkin-instrumentation-express/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for zipkin-instrumentation-express 0.11
+// Project: https://github.com/openzipkin/zipkin-js#readme
+// Definitions by: York Yao <https://github.com/plantain-00>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
```

Comment:
Add a comment `// TypeScript Version: 2.2` here (line 5) because `express` requires it.